pkg/client/clientset/versioned/typed/quotasubtree/v1/quotasubtree_client.go

```diff
@@ -19,27 +19,27 @@ limitations under the License.
 package v1
 
 import (
-	rest "k8s.io/client-go/rest"
 	v1 "github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/apis/quotaplugins/quotasubtree/v1"
-	"github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/client/quotasubtree/clientset/versioned/scheme"
+	"github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/client/clientset/versioned/scheme"
+	rest "k8s.io/client-go/rest"
 )
 
-type QuotaSubtreeV1Interface interface {
+type IbmV1Interface interface {
 	RESTClient() rest.Interface
 	QuotaSubtreesGetter
 }
 
-// QuotasubtreeV1Client is used to interact with features provided by the quotasubtree group.
-type QuotasubtreeV1Client struct {
+// IbmV1Client is used to interact with features provided by the ibm.com group.
+type IbmV1Client struct {
 	restClient rest.Interface
 }
 
-func (c *QuotasubtreeV1Client) QuotaSubtrees(namespace string) QuotaSubtreeInterface {
+func (c *IbmV1Client) QuotaSubtrees(namespace string) QuotaSubtreeInterface {
 	return newQuotaSubtrees(c, namespace)
 }
 
-// NewForConfig creates a new QuotasubtreeV1Client for the given config.
-func NewForConfig(c *rest.Config) (*QuotasubtreeV1Client, error) {
+// NewForConfig creates a new IbmV1Client for the given config.
+func NewForConfig(c *rest.Config) (*IbmV1Client, error) {
```

Comment:
Nit: replace `IbmV1Client` to `CfV1Client`

Reply:
@asm582 This file is not hand generated and is generated by the tool. It makes no sense to edit a generated file.

Reply:
I think it's been driven by the API group name, so it'll change as part of #341.